### PR TITLE
added codegen for  java OK Http Client

### PR DESCRIPTION
**Pull request by shalika10**

### CODE REVIEW

**Review by liyasthomas — APPROVED** [review]

**Comment by shalika10** [comment]
@liyasthomas  thanks !!

**Comment by liyasthomas** [comment]
@shalika10 just noticed one thing. It would be great if you could move Java OK HTTP Client entry right next to JavaScript entry. We maintain alphabetical order in that list.

File: helpers/codegen/codegen.js [hunk 1]
```diff
@@ -42,6 +43,7 @@ export const codegens = [
   SalesforceApexCodegen,
   ShellHTTPie,
   WgetCodegen,
+  JavaOkHttpClientCodegen,
```
Comment (liyasthomas) [inline helpers/codegen/codegen.js:46]: move Java OK HTTP Client to `JSXHRCodegen`

Comment (shalika10) [reply helpers/codegen/codegen.js:46]: sure @liyasthomas

Comment (shalika10) [reply helpers/codegen/codegen.js:46]: raising PR again after the changes

**Review by liyasthomas — APPROVED** [review]
LGTM. Thanks for the quick PR.

File: helpers/codegen/codegen.js (hunk 1 above)
Comment (shalika10) [reply helpers/codegen/codegen.js]: @liyasthomas pushed latest changed kindly merge PR now

**Comment by shalika10** [comment]
@liyasthomas  do we need to have this hacktoberfest accepted tag in PR ??

**Comment by liyasthomas** [comment]
There's no need for that once it's merged. Anyways I'll add the tag.

**Comment by shalika10** [comment]
> There's no need for that once it's merged. Anyways I'll add the tag.

ohh i am new to Open source contribution its my second PR thats why asked !! thanks between

**Pull request merged**